src/login/logind-dbus.c

```diff
@@ -2309,6 +2309,24 @@ static int setup_wall_message_timer(Manager *m, sd_bus_message* message) {
         return 0;
 }
 
+static int cache_wakeup_count(void) {
+        static const char kernel_wakeup_count[] = "/sys/power/wakeup_count";
+        static const char systemd_wakeup_count[] = "/run/wakeup_count";
```

Comment:
honetsly, i'd just specify the paths directly in the read_full_virtual_file() parameters, instead of via this static/const array. It's just more readable that way.